locales/fr.json

```diff
@@ -434,5 +434,6 @@
   "St. Catharines-Niagara": "St. Catharines-Niagara",
   "Sudbury": "Sudbury",
   "Thunder Bay": "Thunder Bay",
-  "Windsor": "Windsor"
-}
+  "Windsor": "Windsor",
+  "Did you pay property tax?": "Did you pay property tax?"
```

Comment:
Let's load the page in french with the other key as well otherwise it will hang around for later prs.